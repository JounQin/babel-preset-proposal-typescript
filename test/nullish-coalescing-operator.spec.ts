import _ from './nullish-coalescing-operator'
import { execute } from './helpers'

const proposal = 'nullish-coalescing-operator'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_()).toEqual([
    'some other default',
    'some other default',
    '',
    0,
    false,
  ])
})
