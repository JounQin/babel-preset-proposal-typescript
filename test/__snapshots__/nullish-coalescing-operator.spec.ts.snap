// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nullish-coalescing-operator 1`] = `"Unexpected token '.'"`;
